src/lib/createPollMessage.ts

```diff
@@ -36,9 +36,16 @@ export async function createPollMessage(data: IUIKitViewSubmitIncomingInteractio
     }
 
     if (options.length === 1) {
-        throw {
-            'option-1': 'Please provide one more option',
-        };
+        if (!state.poll['option-0'] || (state.poll['option-0'] && state.poll['option-0'] === '')) {
```

Comment:
you don't need to test the existence of `state.poll['option-0']` two times.. so you could change to this
```suggestion
        if (!state.poll['option-0'] || state.poll['option-0'] === '') {
```